### ISSUE TITLE
Should we keep a generalized leaderboard for groups?

### ISSUE THREAD

**Issue by Chovin** [FreeCodeCampGuam/freecodecampguam.github.io#21]
our original idea was to make this generalized so that other local groups could clone/fork ours to use for their own groups with minimum hassle.

once we devise a system, should we continue to focus on making it generalized?
It may just end up being a backend api that they can fork and set up real quick.
Since this may become kinda complicated with possible authorizations and whatnot, should we start communicating about the leaderboard on FCC's gitter, and see if people are interested in helping or if there's already one in development?

**Comment by ghost** [comment]
Yes, we can keep it generalized.

**Comment by ghost** [comment]
Yes, we can keep it generalized.

**Comment by jasperdanan** [comment]
yes please. we would like to have one in our local group, FCCG@JFK
